account_asset_management/models/account_asset.py

```diff
@@ -626,11 +626,11 @@ def _get_fy_duration(self, fy, option='days'):
                         duration = (fy_date_stop - fy_date_start).days + 1
                     else:
                         duration = (
-                            datetime(year, 12, 31) - fy_date_start).days + 1
+                            datetime(year, 12, 31).date() - fy_date_start).days + 1
```

Comment:
```suggestion
                            date(year, 12, 31) - fy_date_start).days + 1
```

Reply:
Don't forget to update the import statement to import date
`from datetime import date,datetime`